README.md

```diff
@@ -1,39 +1,57 @@
 # Assessing your JavaScript Fundamentals
-This sprint you worked really hard on learning the fundamentals of JavaScript! This sprint challenge will touch on each concept covered throughout this week.  Follow each task in order but feel free to skip something if you're struggling and come back to it later.
 
-* This project is to be worked on alone but you can use outside resources. You can _reference_ any old code you may have and the training kit content.
+This sprint you worked really hard on learning the fundamentals of JavaScript! This sprint challenge will touch on each concept covered throughout this week. Follow each task in order but feel free to skip something if you're struggling and come back to it later.
+
+- This project is to be worked on alone but you can use outside resources. You can _reference_ any old code you may have and the training kit content.
 
 ## Task 1: Set Up The Project With Git
 
-* [ ] Fork the project into your GitHub user account
-* [ ] Clone the forked project into a directory on your machine
-* [ ] You are now ready to build this project with your preferred IDE
+- [x] Fork the project into your GitHub user account
+- [x] Clone the forked project into a directory on your machine
+- [x] You are now ready to build this project with your preferred IDE
 
 ## Task 2: Comprehension Questions
-Answer the following questions to the best of your ability. You can exercise your Googling skills and use training kit.  Open up the [Answers.md](Answers.md) file and record your responses there.
 
-1. Describe the biggest difference between `.forEach` & `.map`.
-2. What is the difference between a function and a method?
-3. What is closure?
+Answer the following questions to the best of your ability. You can exercise your Googling skills and use training kit. Open up the [Answers.md](Answers.md) file and record your responses there.
+
+1. Describe the biggest difference between `.forEach` & `.map`: forEach applies to every item in an array and doesn't return anything, map returns an array of the same size.
+
```

Comment:
The biggest takeaway from that should be:
forEach manipulates the array while map makes a copy.
there will be instances where you don't want to mess with the original